libs/menu-matriarch/settings/feature/src/lib/settings.component.ts

```diff
@@ -16,7 +16,7 @@ import {
   CardComponent,
   InputComponent,
   SectionComponent,
-} from '@atocha/menu-matriarch/shared/ui-generic';
+} from '@atocha/menu-matriarch/shared/ui';
```

Comment:
This component can use the `InlineDaySelectComponent` with some refactoring in the next PR. The form will have to patch values received from its output manually

Reply:
Decided to leave this alone for now